pkg/server/backuprestoreserver.go

```diff
@@ -196,6 +201,9 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 					b.logger.Info("backup-restore stops leading...")
 				}
 				handler.SetSnapshotterToNil()
+
+				// To do: For Multi-node etcd HTTP status need to be set to `StatusServiceUnavailable` only when backup-restore is in "StateUnknown".
```

Comment:
Why don't you implement the TODO now? Single node should never be `StateUnknown`, right? So, if single node is somehow in `StateUnknown` we can safely set HTTP status for StatusServiceUnavailable . Or am I missing something?

Reply:
>Single node should never be StateUnknown, right?

actually no, when `single node etcd` goes down/crash then backup-restore can go to `StateUnknown` from `StateLeader`.

> Why don't you implement the TODO now? 

yeah, ideally setting the HTTP status to `StatusServiceUnavailable` should be [here](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/leaderelection/leaderelection.go#L87-L105) when backup-restore goes to `StateUnknown` from any state but to do this [LeaderElector struct](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/leaderelection/leaderelection.go#L46-L55)  should have [HTTP Handler object](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/server/httpAPI.go#L68). I wasn’t sure to do this, so for time being and for single node etcd I choose to go with above method since  single node `etcd` backup-restore can only be either in `StateLeader ` or in `StateUnknown`.

Reply:
Okay. Got it. Thanks.